smart-city-electron-dashboard/src/server/server.js

```diff
@@ -33,6 +34,65 @@ mongoose.connect(process.env.MONGODB_URI, {
 // API Routes
 app.use('/api/auth', authRoutes);
 
+// Serve the React frontend files (important for production)
+if (process.env.NODE_ENV === 'production') {
+  app.use(express.static(path.join(__dirname, '../client/build')));
+
+  app.get('*', (req, res) => {
+    res.sendFile(path.join(__dirname, '../client/build', 'index.html'));
+  });
+}
+// MongoDB connection
+const client = new MongoClient(process.env.MONGODB_URI || 'mongodb+srv://SCT:data298B@cluster0.fcat7.mongodb.net/?retryWrites=true&w=majority', {
```

Comment:
Dont' hard code db url here. Please get the uri from env file. check other pages where we initialize mongodb. the purpose server.js is just to connect to mongodb.

Move all the db querying logic to auth.js in server folder.